src/WooCommerce.php

```diff
@@ -552,11 +563,20 @@ public static function woocommerce_display_item_meta($html, $item, $args) {
     }
 
     foreach ($product_data_set as $productData) {
-      $strings[] = '<strong class="wc-item-meta-label">' . $productData['name'] . ':</strong> ' . $productData['value'];
+      $string = '';
+      if (!empty($productData['name'])) {
```

Comment:
Does it make sense to output a heading if there is no value?

Shouldn't it rather be negated?  If there is no heading, skip to the next item?

Reply:
I introduced this for the separator, as I want to display a <hr> without any preceding value. But after checking, the hr is stripped and an empty <li> is rendered. But as we spent quite some time already, I decided to leave it and address this at a later point.